data/json/scenarios.json

```diff
@@ -722,7 +722,7 @@
     "id": "foodplace",
     "name": "The Mascot Rises",
     "points": 0,
-    "description": "You just finished your shift and got back in the break room when you heard the alarms and the security door shutting down behind you.  There's a lot of hungry customers out there and you're not sure Foodplace's delicious food™ is going to be enough to satisfy them.",
+    "description": "Business had been bad—for some reason, weeks of rioting, an evacuation order, and the sky breaking with alien shadows prompted a downturn in foot traffic, even among Food Place’s chain restaurants.  As an employee, upon evacuation being declared, you disregarded company orders to lock up your outlet and, along with another survivor, opted to close yourselves within the breakroom rather than braving the streets awash with insanity and blood.  Safe, for now, behind a security door, you’re convicted that, at least, you’re not going to die in a cheep fast food joint.",
```

Comment:
```suggestion
    "description": "Business had been bad—for some reason, weeks of rioting, an evacuation order, and the sky breaking with alien shadows prompted a downturn in foot traffic, even among Foodplace’s chain restaurants.  As an employee, upon evacuation being declared, you disregarded company orders to lock up your outlet and, along with another survivor, opted to close yourselves within the breakroom to recuperate and plan your next move. Will Foodplace's delicious food™ be enough to placate the horde of hungry customers banging at the door?",
```
`Foodplace` is one word, I also changed the last sentence to keep some of levity/stupidity of the original description